File: src/main/java/me/zeroeightsix/kami/command/commands/BaritoneCommand.java
package me.zeroeightsix.kami.command.commands;

import baritone.Baritone;
import baritone.api.BaritoneAPI;
import me.zeroeightsix.kami.command.Command;

import static me.zeroeightsix.kami.util.MessageSendHelper.sendChatMessage;

/**
 * Created by Dewy on the 17th of April, 2020
 */
public class BaritoneCommand extends Command {

    public BaritoneCommand() {
        super("baritone", null, "b");
        setDescription("Runs baritone commands!");
    }

    @Override
    public void call(String[] args) {
        StringBuilder command = new StringBuilder();
        for (String arg : args) {
            command.append(" ").append(arg);
        }
        // TODO: leij dms 
//        BaritoneAPI.getProvider().getPrimaryBaritone().getCommandManager().execute()
    }
}

File: src/main/java/me/zeroeightsix/kami/command/commands/BaritoneCommand.kt
package me.zeroeightsix.kami.command.commands

import me.zeroeightsix.kami.command.ClientCommand
import me.zeroeightsix.kami.command.CommandManager
import me.zeroeightsix.kami.command.CommandOld
import me.zeroeightsix.kami.command.syntax.ChunkBuilder
import me.zeroeightsix.kami.command.syntax.parsers.EnumParser
import me.zeroeightsix.kami.util.text.MessageSendHelper

object BaritoneCommand : ClientCommand(
    name = "Baritone",
    alias = arrayOf("b")
) {
    init {
        greedy("arguments") { args ->
            execute {
                val newArgs = CommandManager.tryParseArgument(args.value)?: return@execute
                MessageSendHelper.sendBaritoneCommand(*newArgs)
            }
        }
    }

}